requirements.txt

```diff
@@ -0,0 +1,2 @@
+python-dotenv==0.21.0
```

Comment:
yeah, these two are probably sufficient for now.  As you add in other required bits that you interact with (outside the standard library) add them in.  So if you bring in spacy, definitely lock that version in here.  Same if you bring in a graphics library to plot something.